com.unity.ml-agents/Tests/Editor/Sensor/SensorShapeValidatorTests.cs

```diff
@@ -0,0 +1,145 @@
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace MLAgents.Tests
+{
+    public class NoopSensor : ISensor
```

Comment:
What is a Noop?

Reply:
https://en.wikipedia.org/wiki/NOP_(code)

Reply:
I can rename to DummySensor or something clearer

Reply:
NoOpSensor?